pkg/integrations/statsd_exporter/statsd_exporter.go

```diff
@@ -125,7 +125,10 @@ func New(log log.Logger, c *Config) (integrations.Integration, error) {
 	if c.ListenUDP == "" && c.ListenTCP == "" && c.ListenUnixgram == "" {
 		return nil, fmt.Errorf("at least one of UDP/TCP/Unixgram listeners must be used")
 	}
-	statsdMapper := &mapper.MetricMapper{MappingsCount: m.MappingsCount}
+	statsdMapper := &mapper.MetricMapper{
+		MappingsCount: m.MappingsCount,
+		Logger:        log,
+	}
```

Comment:
Are there any other fields this needs to have filled out so it doesn't panic? The struct exports quite a bit.

Reply:
Nope, these are the only fields statsd_exporter fills out.